posts/2021-11-30-microprofile-5-22001-beta.adoc

```diff
@@ -0,0 +1,203 @@
+
+For 22.0.0.1-beta, there is a single beta package for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: this package contains all Open Liberty beta features and GA features and functions.
+
+This means that you can try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you give the beta package a try, <<feedback, let us know what you think>>.
+
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features and enhancements:
+
+* <<mp_5.0, MicroProfile 5.0>>
+* <<rest_client, MicroProfile Rest Client 3.0>>
+* <<jpa_persistence, Define JPA persistence properties at server scope>>
+
+
+[#mp_5.0]
+=== MicroProfile 5.0
+
+MicroProfile 5.0 enables applications to use MicroProfile APIs together with Jakarta EE 9.1. MicroProfile 5.0 does not provide any other functional updates except aligning with Jakarta EE 9.1.  MicroProfile 5.0 includes Config 3.0, Fault Tolerance 4.0, Rest Client 3.0, Health 4.0, Metrics 4.0, Open Tracing 3.0, Open API 3.0, and JWT propagation 2.0. This beta driver will be used as a compatible implementation for releasing MicroProfile 5.0.
+
+Include the following in the `server.xml` to enable the MicroProfile 5.0 feature in Open Liberty:
+
+[source, xml]
+----
+<featureManager>              
+    <feature>microProfile-5.0</feature>
+</featureManager>
+----
+
+Alternatively, you can enable the individual MicroProfile features that you need, for example:
+
+[source, xml]
+----
+<featureManager>              
+    <feature>mpOpenTracing-3.0</feature>
+</featureManager>
+----
+
+For more information about this update check out the link:https://github.com/eclipse/microprofile/releases/tag/5.0[MicroProfile 5.0 Release] on GitHub.
+
+
+[#rest_client]
+=== MicroProfile Rest Client 3.0
+
+MicroProfile Rest Client is an API intended to enable developers to write type-safe interfaces to abstract and invoke remote RESTful services. This is the 3.0 release of MicroProfile Rest Client, and it adds support for Jakarta EE 9.1 technologies - from a developer's perspective, the only change from the previous release (2.0) is the package space name change of Jakarta packages from `javax.*` to `jakarta.*`. Another change is that the Open Liberty implementation has changed from Apache CXF to RESTEasy - this change brings with it some behavior and property changes (most of which are already documented as link:{url-prefix}/docs/latest/reference/jaxrs-dif.html[differences between `jaxrs-2.1` and `restfulWS-3.0`]).  
+
+To use this new feature, you would need to add `mpRestClient-3.0` to the `featureManager` element in the server.xml. The code should be similar to previous versions of MP Rest Client, but the packages should change from `javax.*` to `jakarta.*`.
```

Comment:
you would need to add
--->
add

Reply:
reference here to "the code" seems a little vague- would "your application code" be correct? If so, recommend

Your application code can be similar to previous versions of MP Rest Client, but the package names must change from `javax.*` to `jakarta.*`.

Reply:
check javax.* to jakarta.*.